scripts/xpost.coffee

```diff
@@ -16,8 +16,8 @@
 
 module.exports = (robot) ->
   console.log("XPOST script loaded.")
-  robot.hear /\bxpost #([\w\-]+)/i, (msg) ->
-    target = msg.match[1]
+  robot.hear /\bx(\-)?post #([\w\-]+)/i, (msg) ->
```

Comment:
FYI you don't actually need the parenthesis, and then you could leave it as the first match below.